readthedocs/builds/models.py

```diff
@@ -693,6 +694,9 @@ class Meta:
             ['version', 'state', 'type'],
             ['date', 'id'],
         ]
+        indexes = [
+            models.Index(fields=['project', 'date']),
+        ]
```

Comment:
I think this is not required if you are already using `db_index=True` in the fields. https://docs.djangoproject.com/en/3.2/ref/models/options/#indexes

Reply:
This is indexing them together.